kotest-common/api/kotest-common.api

```diff
@@ -14,6 +14,21 @@ public abstract interface annotation class io/kotest/common/JVMOnly : java/lang/
 public abstract interface annotation class io/kotest/common/KotestInternal : java/lang/annotation/Annotation {
 }
 
+public final class io/kotest/common/NonConstantResultsKt {
+	public static final fun nonConstantFalse ()Z
+	public static final fun nonConstantTrue ()Z
+}
+
+public final class io/kotest/common/NonDeterministicTestVirtualTimeEnabled : kotlin/coroutines/CoroutineContext$Element, kotlin/coroutines/CoroutineContext$Key {
```

Comment:
`NonDeterministicTestVirtualTimeEnabled` is marked as `@KotestInternal`, so I'm confused that it's here. Shouldn't it be excluded from the API dump? 🤔

Reply:
The Kotlin BCV (`apiDump`/`apiCheck`) does not know about `@KotestInternal`. That could be changed by [configuring the BCV](https://github.com/Kotlin/binary-compatibility-validator?tab=readme-ov-file#optional-parameters), but would be beyond the scope of this PR.

Reply:
Got it, thanks!